### ISSUE TITLE
Unable to set boolean value to False

### ISSUE THREAD

**Issue by b-jazz** [bw2/ConfigArgParse#35]
If I have a boolean with a default of True, it appears impossible to set it to False from the command line since you can only set it to true with a `--foobar` singleton and you can't set it to false with an environment variable either.

    (cap) /tmp $ cat ./cap.py
    #!/usr/bin/env python
    
    import configargparse
    
    parser = configargparse.ArgParser()
    parser.add('--foobar', action='store_true', default=True)
    args = parser.parse_args()
    parser.print_values()
    if args.foobar:
        print('foobar is True')
    else:
        print('foobar is False or undefined')
    (cap) /tmp $ ./cap.py
    foobar is True
    (cap) /tmp $ export foobar=False
    (cap) /tmp $ ./cap.py
    foobar is True
    (cap) /tmp $ export FOOBAR=False
    (cap) /tmp $ ./cap.py
    foobar is True
    (cap) /tmp $ echo $FOOBAR $foobar
    False False

**Comment by bw2** [comment]
What about doing
`parser.add('--foobar', action='store_false', default=True)`
instead of
`parser.add('--foobar', action='store_true', default=True)`
?

**Comment by b-jazz** [comment]
Using `store_false` would necessitate the changing of the name of the variable which would be awkward in the particular CLI that I'm writing. It is a workaround, but not as desirable as being able to put `Foobar=False` in the config file.

(Also, I realized just now that I didn't put an env_var while doing the `add` of the parameter, but that doesn't really "fix" my situation. It is still impossible to force a flag off or to false.)

I just did another test of using `action='store_true'` and setting it to True in the config, but then trying to unset or override to False with an environment variable, and it can't be done. It would be nice to be able to do so.
